src/main/java/com/faforever/api/user/UserService.java

```diff
@@ -25,23 +27,25 @@
 @Slf4j
 public class UserService {
 
-  private static final Pattern USERNAME_PATTERN = Pattern.compile("[A-Za-z][A-Za-z0-9_-]{2,15}$");
   static final String KEY_ACTION = "action";
   static final String KEY_EXPIRY = "expiry";
   static final String KEY_USERNAME = "username";
   static final String KEY_EMAIL = "email";
   static final String KEY_PASSWORD = "password";
+  static final String KEY_USER_ID = "id";
```

Comment:
This case is no dynamic data, it's constants for the keys of the token. I don't see why this should be in another class and clunk up space.